tokio-util/src/time/mod.rs

```diff
@@ -8,45 +8,18 @@
 //!
 //! This type must be used from within the context of the `Runtime`.
 
-use std::future::Future;
 use std::time::Duration;
-use tokio::time::Timeout;
 
 mod wheel;
 
 pub mod delay_queue;
 
+// re-export `FutureExt` to avoid breaking change
+pub use crate::future::FutureExt;
```

Comment:
```suggestion
// re-export `FutureExt` to avoid breaking change
#[doc(inline)]
pub use crate::future::FutureExt;
```

<img width="2540" height="786" src="https://github.com/user-attachments/assets/39bbab57-afd6-40f9-abe8-327b15aaab7b" />